README.md

```diff
@@ -79,9 +106,9 @@ Creating a task manifest file.
                     "plugin_name": "statistics",
```

Comment:
Line 96: I think "/intel/mock/*/baz" are string metrics. Will not work with the plugin. (maybe I'm wrong, you can test if the task work)